kie-server-parent/kie-server-tests/kie-server-integ-tests-common/src/main/java/org/kie/server/integrationtests/shared/basetests/RestJmsSharedBaseIntegrationTest.java

```diff
@@ -47,16 +50,14 @@
     @Parameterized.Parameters(name = "{index}: {0} {1}")
     public static Collection<Object[]> data() {
         KieServicesConfiguration restConfiguration = createKieServicesRestConfiguration();
-
         Collection<Object[]> parameterData = new ArrayList<Object[]>(Arrays.asList(new Object[][]
                         {
                                 {MarshallingFormat.JAXB, restConfiguration},
-                                {MarshallingFormat.JSON, restConfiguration},
+                                {MarshallingFormat.JSON,  createKieServicesRestConfiguration(Collections.singletonMap(KieServerConstants.KIE_CONTENT_TYPE_HEADER, "application/json; fields=not_null"))},
```

Comment:
This will make this very slow at integration test level (veeeeeeeeery slow).  and you are testing the full logic of marshalling not really checking whether those fields are being retrieved or not.

You need to check directly against the kie-server and see the string response whether thoses fields are returned o not.

Reply:
That nulls are not returned when not null are specified is already tested in Unit test. The purposse of modifiying integration test is to test that the entity is still created when null parameters are skipped.

Reply:
I think that we should keep the default settings in here.
Testing of skipped null fields could be done in some specific integration test only, for example something like  https://github.com/kiegroup/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/rest/ProcessServiceRestOnlyIntegrationTest.java

Reply:
Added new test class under specified directory. Thanks for the advice